docs/source/Cube Helper Tutorial.rst

```diff
@@ -370,3 +370,69 @@ This now seems to have worked, Lets have a look:
               mean: time (3 hours)
 
 Success!
+
+``cube_helper`` can load iris cubes without any output, with the ``muffle_logger`` function.
+This should only be used when it is absolutely necessary.
+
+.. code:: ipython3
+
+    ch.muffle_logger()
+    cube = ch.load('/path/to/cmip/data/HadGEM3-GC31-LL/piControl/r1i1p1f1/Amon/tasmin/gn/v20190628')
+
+
+.. parsed-literal::
+
+    /opt/scitools/environments/default/current/lib/python3.6/site-packages/iris/fileformats/cf.py:798: UserWarning: Missing CF-netCDF measure variable 'areacella', referenced by netCDF variable 'tasmin'
+      warnings.warn(message % (variable_name, nc_var_name))
+    /opt/scitools/environments/default/current/lib/python3.6/site-packages/iris/fileformats/cf.py:798: UserWarning: Missing CF-netCDF measure variable 'areacella', referenced by netCDF variable 'tasmin'
+      warnings.warn(message % (variable_name, nc_var_name))
+    /opt/scitools/environments/default/current/lib/python3.6/site-packages/iris/fileformats/cf.py:798: UserWarning: Missing CF-netCDF measure variable 'areacella', referenced by netCDF variable 'tasmin'
+      warnings.warn(message % (variable_name, nc_var_name))
+    /opt/scitools/environments/default/current/lib/python3.6/site-packages/iris/fileformats/cf.py:798: UserWarning: Missing CF-netCDF measure variable 'areacella', referenced by netCDF variable 'tasmin'
+      warnings.warn(message % (variable_name, nc_var_name))
+    /opt/scitools/environments/default/current/lib/python3.6/site-packages/iris/fileformats/cf.py:798: UserWarning: Missing CF-netCDF measure variable 'areacella', referenced by netCDF variable 'tasmin'
+      warnings.warn(message % (variable_name, nc_var_name))
+    /opt/scitools/environments/default/current/lib/python3.6/site-packages/iris/fileformats/cf.py:798: UserWarning: Missing CF-netCDF measure variable 'areacella', referenced by netCDF variable 'tasmin'
+      warnings.warn(message % (variable_name, nc_var_name))
+    /opt/scitools/environments/default/current/lib/python3.6/site-packages/iris/fileformats/cf.py:798: UserWarning: Missing CF-netCDF measure variable 'areacella', referenced by netCDF variable 'tasmin'
+      warnings.warn(message % (variable_name, nc_var_name))
+    /opt/scitools/environments/default/current/lib/python3.6/site-packages/iris/fileformats/cf.py:798: UserWarning: Missing CF-netCDF measure variable 'areacella', referenced by netCDF variable 'tasmin'
+      warnings.warn(message % (variable_name, nc_var_name))
```

Comment:
I would remove these lines. If you had areacella available then these wouldn't show. Perhaps, could you make the same change elsewhere in the documentation please?